docs/content/preview/deploy/multi-dc/async-replication/includes/automatic-setup.md

```diff
@@ -63,9 +63,7 @@ The following assumes you have set up Primary and Standby universes. Refer to [S
     ./yugabyted restore --cloud_storage_uri <AWS/GCP/local cloud storage uri>  --database <database_name> --base_dir <base_dir of target node>
     ```
 
-1. If needed, perform a full copy of the database(s) on the Primary to the Standby using distributed [backup](../../../../reference/configuration/yugabyted/#backup) and [restore](../../../../reference/configuration/yugabyted/#restore).
-
-    Note that if your source database is not empty, it must be bootstrapped, even if the output suggests otherwise. This applies even if it contains only empty tables, unused types, or enums (issue [#24030](https://github.com/yugabyte/yugabyte-db/issues/24030)).
+1. Perform a full copy of the database(s) on the Primary to the Standby using distributed [backup](../../../../reference/configuration/yugabyted/#backup) and [restore](../../../../reference/configuration/yugabyted/#restore).
```

Comment:
add a note that this is essential. Just running the same DDLs on the target wont work even if the source does not have any data since oids, and other internal stuff needs to be the same on both sides